docs/user_guide/source/advice/advice.rst

```diff
@@ -87,4 +87,8 @@ The goal is to provide specific advice and good practices about the use of ADIOS
 25. Developers: explore the testing infrastructure ``ADIOS2/testing`` in ADIOS 2 as a starting point for using ADIOS 2 in your own testing environment. 
 
 26. Become a super-user of :ref:`bpls : Inspecting Data` to analyze datasets generated by ADIOS 2.
+
+27. When you need to write intermediate results, such as checkpoints, considering streaming your data instead of writing
```

Comment:
Not sure I agree with this for checkpointing. Where do you stream the data and how would you recover a failure if the data is not on the permanent storage?